plugins/channelrx/channelpower/channelpowersink.cpp

```diff
@@ -21,12 +21,13 @@
 #include "channelpowersink.h"
 
 ChannelPowerSink::ChannelPowerSink(ChannelPower *channelPower) :
-        m_channelPower(channelPower),
```

Comment:
Indeed m_channelPower is unused. I think it should be either completely removed from the constructor parameters or set to nullptr

Reply:
ok